### DISCUSSION TITLE
Make it explicit that numbers have been hidden.

### DISCUSSION THREAD

**Discussion by harishbsrinivas** [ananthakumaran/paisa#97] [Ideas]
I started using Paisa by entering A few ledger entries recently. The simple input method is wonderful. The ability to keep the details barebones or as granular as possible is much appreciated. I recently came across an issue that might hit other users as well.

 While playing around with the interface and menus, I enabled hide numbers. I completely forgot about this and fast-forward to a few days later, began importing a lot of data via CSV import. I was puzzled that all the Dashboard, recent posts, Balance etc. were showing zero. Please add a banner or some sort of visual indicator to show that the "Hide Numbers" Option is enabled and that is the reason "0" is shown everywhere.

**Comment by ananthakumaran** [comment]
@harishbsrinivas I am considering altering the paisa logo at the top left when "hide numbers" option is enabled like incognito mode in browser (the changes are available at  https://demo.paisa.fyi/). Let me know what you think.
